tests/Unit/ExampleTest.php

```diff
@@ -3,6 +3,7 @@
 namespace Tests\Unit;
 
 use PHPUnit\Framework\TestCase;
+use App\Models\User;
```

Comment:
Remove this line

Reply:
the file with that namespace in my project does not exist...